File: C++/program149.cpp
#include<iostream>
using namespace std;

bool CheckCapital(char ch)
{
    bool bFlag = false;
    if((ch >= 65 ) && (ch <= 90))
    {
       bFlag = true;
    }
    return bFlag;
  
}

int main()
{
    char cValue = '\0';
    bool bRet = false;

    cout<<"Enter charcter"<<'\n';
    cin>>cValue;

    bRet = CheckCapital(cValue);

    if(bRet == true)
    {
        cout<<"charcter is capital"<<'\n';
    }
    else
    {
        cout<<"charcter is not capital"<<'\n';
    }
    return 0;
}

File: Java/program149.java
import java.util.*;

class CharcterProgram
{
    public boolean CheckSmall(char ch)
    {
        if((ch >= 'a') && (ch <= 'z'))
        {
            return true;
        }
        else
        {
            return false;
        }

    }
}

class program149
{
    public static void main(String A[])
    {
        char cValue =' ';
        boolean bRet = false;
        
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter charcter :");
        cValue = sobj.next().charAt(0);

        CharcterProgram cobj = new CharcterProgram();
        bRet = cobj.CheckSmall(cValue);

        if(bRet == true)
        {
            System.out.println(cValue+" is small letter");
        }
        else
        {
            System.out.println(cValue + " is not small letter");
        }

     
         

    }
}